spring-cloud-huawei-dependencies/pom.xml

```diff
@@ -177,6 +178,11 @@
         <artifactId>spring-context-support</artifactId>
         <version>${alibaba.context.support.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.json</groupId>
```

Comment:
Maybe do not need gson and json library, just use jackson.

Reply:
fixed